scripts/compile-binaries.sh

```diff
@@ -162,6 +162,15 @@ compile_prusa_buddy() {
     cp /home/pi/klipper/out/klipper.bin /home/pi/printer_data/firmware_binaries/firmware-prusa-buddy.bin
 }
 
+compile_prusa_einsy() {
+    echo "Compiling firmware for Prusa Einsy"
+    cp -f /home/pi/printer_data/config/RatOS/boards/prusa-einsy/firmware.config /home/pi/klipper/.config
+    make olddefconfig
+    make clean
+    make
+    cp /home/pi/klipper/out/klipper.bin /home/pi/printer_data/firmware_binaries/firmware-prusa-einsy.bin
```

Comment:
change `printer_data` to `printer_data/config`

Reply:
Done